example/src/partials/head.js

```diff
@@ -11,8 +11,6 @@ export default function(title) {
       { /* we might be able to inject these (or supply them as props) */}
       <link href='/index.css' rel='stylesheet' type='text/css' />
       <link href='https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.css' rel='stylesheet' type='text/css' />
-      <script defer src={`https://unpkg.com/react@15.6.1/dist/react${isProduction ? '.min' : ''}.js`} />
-      <script defer src={`https://unpkg.com/react-dom@15.6.1/dist/react-dom${isProduction ? '.min' : ''}.js`} />
       <script defer src={`https://unpkg.com/babel-polyfill@6.23.0/dist/polyfill${isProduction ? '.min' : ''}.js`} />
```

Comment:
Do we still need this?

Reply:
https://medium.com/@jcse/clearing-up-the-babel-6-ecosystem-c7678a314bf3

Reply:
Ok, so it seems the previous link was wrong:

https://babeljs.io/docs/plugins/transform-runtime/
> NOTE: Instance methods such as "foobar".includes("foo") will not work since that would require modification of existing built-ins (Use babel-polyfill for that)

https://babeljs.io/docs/usage/polyfill/
> This means you can use new built-ins like Promise or WeakMap, static methods like Array.from or Object.assign, instance methods like Array.prototype.includes, and generator functions (provided you use the [regenerator plugin](https://babeljs.io/docs/plugins/transform-regenerator/)). The polyfill adds to the global scope as well as native prototypes like String in order to do this.

Reply:
In our case we don't want to worry about features as long as they are in the current spec or explicitly added (#9).

So I guess I'll leave it like it is for now and create an issue to make sure we have it right by default.